config/database.yml

```diff
@@ -5,21 +5,24 @@
 #   gem "sqlite3"
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  username: postgres
+  password: postgres123
 
 development:
   <<: *default
-  database: db/development.sqlite3
-
+  database: db/store-development
+  username: postgres
+  password: postgres123
```

Comment:
Tego chyba nie potrzebujesz tutaj, masz na gorze
```ruby
 <<: *default
 ```
 Więc z automatu tutaj powinno być username i password

Reply:
a to dodałem bo nie pamiętałem, które ostatnio zadziałało xD